assets/css/layouts/header.css

```diff
@@ -5,7 +5,7 @@ header.page-header {
 }
 
 .page-header-inner {
-  /* NOTE: keeps the nav list dropdown a similar width to its corresponding link / list item */
+  /* NOTE: keeps the nav.main-nav list dropdown a similar width to its corresponding link / list item */
```

Comment:
Noting I made the selector for the `nav` element more specific so that it doesn't conflict with other navigation on the page such as the blog pagination.